packages/core/src/types/AABB3.ts

```diff
@@ -0,0 +1,11 @@
+// Axis-aligned bounding box 2D
+type AABB3 = {
```

Comment:
Can we call this MinMax or something like that?  Also, can we re-use one of the existing types?  Or, replace the existing ones with this one?

Reply:
It is the same as the bounding box, but different representation

Reply:
Yeah I think it makes sense to call it bounding box, maybe we do that after the demo. @lscoder